onyo/lib/tests/test_commands_get.py

```diff
@@ -100,21 +101,22 @@ def test_onyo_get_reserved_keys(inventory: Inventory,
                                 capsys) -> None:
     r"""`onyo_get()` allows to specify all reserved keys to query
     and display information for."""
-    from onyo.lib.consts import RESERVED_KEYS, PSEUDO_KEYS
-    reserved = ["type", "make", "model.name", "serial"] + PSEUDO_KEYS + RESERVED_KEYS
+    from onyo.lib.consts import RESERVED_KEYS
+    from onyo.lib.pseudokeys import PSEUDO_KEYS
+    reserved = ["type", "make", "model.name", "serial"] + list(PSEUDO_KEYS.keys()) + RESERVED_KEYS
```

Comment:
Oh ugh. I remember this.

Something we will be sure to unroll by the end of all of this pseudo-key work.